.github/workflows/beam_PreCommit_Java_Examples_Dataflow_Java11.yml

```diff
@@ -110,7 +110,7 @@ jobs:
       - name: run javaExamplesDataflowPrecommit script
         uses: ./.github/actions/gradle-command-self-hosted-action
         with:
-          gradle-command: :javaExamplesDataflowPrecommit
+          gradle-command: :runners:google-cloud-dataflow-java:examples:preCommit
```

Comment:
Replacing `javaExamplesDataflowPrecommit` with `:runners:google-cloud-dataflow-java:examples:preCommit` makes it so that [this](https://github.com/Abacn/beam/blob/adjustexampletest/runners/google-cloud-dataflow-java/examples/build.gradle#L172-L175) runs instead of [this](https://github.com/Abacn/beam/blob/adjustexampletest/build.gradle.kts#L557-L561)?

Reply:
yes this is intended. Commented on description

"Examples PreCommit Dataflow v1 Java11/17 now only run wordcount, which suffices to test java version"

If still run `:javaExamplesDataflowPrecommit` then there are significantly more Dataflow pipeline to run on each PR commit push, then people needs to wait for more time to have test results.